main.tf

```diff
@@ -0,0 +1,75 @@
+resource "aws_route53_zone" "this" {
+  name              = var.name
+  comment           = "Route53 hosted zone"
+  delegation_set_id = var.vpc == null ? var.delegation_set_id : null
+  force_destroy     = var.force_destroy
+  dynamic "vpc" {
+    for_each = var.vpc
+    content {
+      vpc_id = vpc.value.vpc_id
+      #vpc_region = lookup(vpc.value, "vpc_region", null)
```

Comment:
Just curious why this line is commented out

Reply:
That was part of troubleshooting haha, shouldn't be commented out